cmd/influx/user_resource_mapping.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/influxdata/influxdb"
+	"github.com/influxdata/influxdb/cmd/influx/internal"
+)
+
+func membersAddF(urm influxdb.UserResourceMapping, flgs Flags) error {
+	mappingSvc, err := newUserResourceMappingService(flgs)
+	if err != nil {
+		return fmt.Errorf("failed to initialize members service client: %v", err)
+	}
+
+	if err = mappingSvc.CreateUserResourceMapping(context.Background(), &urm); err != nil {
+		return fmt.Errorf("failed to add member: %v", err)
+	}
+	writeUserResourceMapping(urm)
+	return nil
+}
+
+func membersRemoveF(resourceID, userID influxdb.ID, flgs Flags) error {
+	mappingSvc, err := newUserResourceMappingService(flgs)
+	if err != nil {
+		return fmt.Errorf("failed to initialize members service client: %v", err)
+	}
+	if err = mappingSvc.DeleteUserResourceMapping(context.Background(), resourceID, userID); err != nil {
+		return fmt.Errorf("failed to remove member: %v", err)
+	}
+	return nil
+}
+
+func writeUserResourceMapping(urm influxdb.UserResourceMapping) {
```

Comment:
I'd prefer just a printf saying that the user has been added as a member or owner. It's confusing/unhelpful to expose system internals that users don't care about.